openstack/cronus/templates/cronus/_config.yaml.tpl

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.cronus.enabled -}}
+cronus:
+  cacheSize: {{ .Values.cronus.cacheSize }}
+  billingCacheTTL: {{ .Values.config.billingCacheTTL }}
+  aws:
+    forwardUserAgent: {{ .Values.config.forwardUserAgent }}
+    allowedServices:
+      email: {{ .Values.config.allowedServices.email }}
```

Comment:
What if we need to add another stateless service? I suggest:
```
allowedServices: {{ .Values.config.allowedServices }}
```

Reply:
I added a range 
allowedServices:
    {{- range $key, $value := .Values.config.allowedServices }}
      {{ $key }}: {{ $value }}
    {{- end }}